css/style.css

```diff
@@ -1,3 +1,11 @@
+.page {
+    display: center;
+    flex-direction: column;
```

Comment:
Since you have only one element the flex-direction: column can be removed and with the other propreties, the view will be the same.